package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rebilly-js-sdk",
-  "version": "27.0.0",
+  "version": "27.1.0",
```

Comment:
Shouldn't this be `28.0.0` since it removes something which is a breaking change?

Reply:
I believe major version should be updated

Reply:
I just updated to 27.0.0 some hours ago with another breaking change so that's why I didn't update to a major. But you are right. I can do that.